e2e/js_binary_workspace/.bazelignore

```diff
@@ -0,0 +1,4 @@
+workspace/bazel-bin
```

Comment:
why is `.bazelignore` necessary? Normally e2e tests only need to list the `node_modules` directories and I don't think we have any?